### PR TITLE
Menu hamburger updated

### PR DESCRIPTION
**Pull request by yvesdaxmaz**
I have made all update which was suggest on the last branch from which I made pull request. I remove the old branch and put all update in this new branch

### CODE REVIEW

File: toc.js [hunk 1]
```diff
@@ -138,4 +138,4 @@ window.addEventListener( 'scroll', function () {
     }
 }, false );
 
-}( document, window ) );
+}( document, window ) );
```
Comment (neilj) [inline toc.js:141]: Please remove this unnecessary change.

**Review by neilj — CHANGES_REQUESTED** [review]
The submodule is still modified in this pull request. Please remove that and the other unnecessary change and I can merge.

File: toc.js (hunk 1 above)
Comment (yvesdaxmaz) [reply toc.js:141]: I have try to remove that line but I realize that line is related to the first ligne in the toc.js file

**Comment by marclaporte** [comment]
This is resolved by new design at https://github.com/jmapio/jmapio.github.io/pull/10 so can be closed.
